python-sdk/src/langwatch/client.py

```
@@ -33,17 +33,28 @@ class Client(LangWatchClientProtocol):
     """
 
+    # Class variables - shared across all instances
+    _instance: Optional["Client"] = None
     _debug: bool = False
-    _api_key: str
-    _endpoint_url: str
-    instrumentors: Sequence[BaseInstrumentor] = []
-    base_attributes: BaseAttributes = {}
+    _api_key: str = ""
+    _endpoint_url: str = ""
+    _base_attributes: BaseAttributes = {}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)